cms/templates/widgets/header.html

```diff
@@ -236,10 +236,11 @@ <h2 class="sr-only">${_("Account Navigation")}</h2>
             <a href="${get_online_help_info(online_help_token)['doc_url']}" title="${_('Contextual Online Help')}" target="_blank">${_("Help")}</a>
           </li>
           <li class="nav-item nav-not-signedin-signup">
-            <a class="action action-signup" href="${reverse('signup')}">${_("Sign Up")}</a>
-          </li>
-          <li class="nav-item nav-not-signedin-signin">
-            <a class="action action-signin" href="${reverse('login')}">${_("Sign In")}</a>
+            <div class="form-actions">
+              <button class="action action-primary" onclick="window.location.href='/auth/login/cms-microsoft-oauth2/?auth_entry=login&amp;next=/home/'">
```

Comment:
Maybe we need to create `onclick` event in the related to this template JS file?
Also, if the backend name will be changed URL will not redirect to the correct endpoint. Do we need to add this part of the URL dynamically from the settings?

Reply:
It’s good, but it was done like microsofts backend https://github.com/raccoongang/microsoft-oauth2-client/blob/master/sso_edx_microsoft/templates/cms/login.html#L22